docs/whatsnew/1.1.rst

```diff
@@ -0,0 +1,182 @@
+.. doctest-skip-all
+
+.. _whatsnew-1.1:
+
+************************
+What's New in SunPy 1.1?
+************************
+
+Overview
+========
+
+The SunPy project is pleased to announce the 1.1 release of the sunpy package.
+
+The headline changes in 1.1 are:
+
+* The `~sunpy.coordinates` subpackage now supports four additional coordinate frames (HCI, HEE, GSE, and GEI).
+* Remote Data Manager (#3124)
+* SUVI support (#3269 / #3301)
+* Initial support for WISPR data (#3501)
+* The import times for `sunpy` and some subpackages are significantly shorter, with no loss of functionality.
+
+On this page, you can read about some of the big changes in this release:
+
+* :ref:`whatsnew-1.1-python`
+* :ref:`whatsnew-1.1-coordinates`
+* :ref:`whatsnew-1.1-dl_manager`
+* :ref:`whatsnew-1.1-SUVI`
+* :ref:`whatsnew-1.1-WISPR`
+* :ref:`whatsnew-1.1-importtime`
+
+SunPy 1.1 also includes a large number of smaller improvements and bug fixes, which are described in the :ref:`changelog`.
+
+By the numbers:
+
+* XX commits have been added since 1.0
+* XX issues have been closed since 1.0
+* XX pull requests have been merged since 1.0
+* XX people have contributed since 1.0
+* XX new contributors
+
+.. _whatsnew-1.1-python:
+
+Supported versions of Python
+============================
+
+SunPy 1.1 comes with support for Python versions 3.6, 3.7 and 3.8.
```

Comment:
Putting this in what's new makes it sound like this support is all new, but probably only support for 3.8 is new?